### ISSUE TITLE
Connection check in multiplexing 

### ISSUE THREAD

**Issue by ilya2204** [ueberdosis/hocuspocus#714]
**The problem I am facing**
When using multiplexing for connecting to a large number of documents via one socket the socket constantly spams with connection check messages (as I understand it is pingInterval) and this happens for each document separately.

**The solution I would like**
Perhaps it can be done so that the connection is checked only for the main socket, and not for each document separately?

**Comment by janthurau** [comment]
hi @ilya2204, this shouldnt happen. I think you are seeing awareness updates, which are updated every 30 seconds per document. If you don't need awareness, you can disable it by passing `awareness: null` to the provider, but make sure that you have at least one provider that uses it (or otherwise receives messages before `HocuspocusProviderWebsocket.messageReconnectTimeout`

**Comment by janthurau** [comment]
I dont think you can raise that as this is yjs internals (see https://docs.yjs.dev/api/about-awareness), but you can just disable it if you don't need it for some (or all) documents.

**Comment by ilya2204** [comment]
Understood thanks. It's just that there are actually different parameters `messageReconnectTimeout`, `forceSyncInterval`, `pingInterval`, `outdatedTimeout`, which are mostly 30 seconds quite confusing.

**Comment by ilya2204** [comment]
I could be wrong, but at first glance, if you copy [Awareness class](https://github.com/yjs/y-protocols/blob/a4dd5de756f1061eaca9c1b4ebb2013e74987a11/awareness.js#L13) and replace the timeout with the desired one, then everything should work.

**Comment by ilya2204** [comment]
@janthurau but it seems that doing ping/pong for each document on the socket, and not for the entire socket as a whole, is an overhead

**Comment by ArihantBapna** [comment]
@ilya2204 were you able to figure out a solution for this?
